src/xenia/gpu/command_processor.cc

```diff
@@ -360,11 +360,11 @@ void CommandProcessor::MakeCoherent() {
   }
 
   const char* action = "N/A";
-  if (status_host & 0x03000000) {
+  if (0x03000000 == (status_host & 0x03000000)) {
```

Comment:
Please flip this so it's `(status_host & ...) == ...`.

Reply:
Should be ok now